keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -1193,11 +1227,13 @@ We're going to better tune the function based on experiments, but based on the
 above back-of-envelope calculations showing that memory should almost never be
 a limiting factor we will apprximate the width simply with:
 ```
-width_approx(n) = min(A, ceil(N / E)), where E = 1 / B
+width_approx(n) = min(A, ceil(N / t_finish_virtual)), where t_finish_virtual = 1 / B
```

Comment:
I'm not following this change - this won't be t_finish_virtual (as it is described above).

Reply:
Oops, that was an over-zealous search-and-replace.

Reply:
Undone in the force-push to 1a58c3eb